scheduler/CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.7.1] - 2017-11-07
+### Added
+- Added support for killing a group of jobs, from @DaoWen
+- Added sysouts to get job output closer to Mesos' CommandExecutor, from @shamsimam
+- Added metrics support for usage of /list, from @dposada
+- Added support for groups on retry endpoint, from @DaoWen
```

Comment:
Maybe "Added support for retrying a group of jobs"?

Reply:
Changed.